lib/livebook_web/live/session_live.ex

```diff
@@ -725,6 +732,60 @@ defmodule LivebookWeb.SessionLive do
             </div>
           <% end %>
         </div>
+
+        <%= if Livebook.Config.feature_flag_enabled?(:hub) do %>
+          <div class="mt-16">
+            <h3 class="uppercase text-sm font-semibold text-gray-500">
+              Enterprise secrets
```

Comment:
Don't we want to manage secrets uniformly, and have enterprise hub as one of the sources? @josevalim

Reply:
This is just a way to see that your secret has been created, because we don't get all secrets on startup.